__tests__/app.test.js

```diff
@@ -84,4 +84,41 @@ describe('GET /api/articles/:article_id', () => {
 				expect(response.body.message).toBe('Invalid article_id');
 			});
 	});
+});
+
+describe('GET /api/articles', () => {
+	test('GET:200 should return an array of articles', () => {
+		return request(app)
+			.get('/api/articles')
+			.expect(200)
+			.then((response) => {
+				const articles = response.body.articles;
+				console.log(articles[1]);
+				expect(articles).toEqual(expect.any(Array));
+				expect(articles.length).toBeGreaterThan(0);
+
+				articles.forEach((article) => {
+					expect(article).toHaveProperty('author');
+					expect(article).toHaveProperty('title');
+					expect(article).toHaveProperty('article_id');
+					expect(article).toHaveProperty('topic');
+					expect(article).toHaveProperty('created_at');
+					expect(article).toHaveProperty('votes');
+					expect(article).toHaveProperty('article_img_url');
+					expect(article).toHaveProperty('comment_count');
+					expect(article).not.toHaveProperty('body');
```

Comment:
By using a jest matcher like toMatchObject you could check all the properties in the object without repeating yourself with toHaveProperty each time. 

You could also add the matcher expect.any() to check the values are strings or numbers for example.

Reply:
done